xml/soc-operations-maintenance.xml

```diff
@@ -482,6 +482,107 @@
     </listitem>
    </itemizedlist>
   </section>
+
+  <section xml:id="sec-upgrade-unsupported-configs">
+   <title>Unsupported configurations</title>
+   <para>
+    In &cloud; 9, certain configurations and barclamp combinations are no
+    longer supported. See the &cloud; 9 release notes for details. The upgrade
+    prechecks fail if they detect an unsupported configuration. Below is a list
+    configurations that cause failure and possible solutions.
+   </para>
+   <variablelist><title>Unsupported Configurations</title>
+  <varlistentry>
+    <term>
+      &aodh; deployed
+    </term>
+    <listitem>
+      <para>
+       The &aodh; barclamp has been removed in &cloud; 9. Deactivate and delete
+       the &aodh; proposal before continuing the upgrade.
+      </para>
+     </listitem>
+  </varlistentry>
+    <varlistentry>
+    <term>
+      &o_meter; deployed without &o_monitor;
+    </term>
+    <listitem>
+      <para>
+       As of &cloud; 9, &o_meter; has been reduced to the &o_meter; agent,
+       with &o_monitor; being used as a storage back-end. Consequently, a
+       standalone &o_meter; without &o_monitor; present will fail the upgrade
+       prechecks. There are two possible solutions.
+      </para>
+      <itemizedlist>
+       <listitem>
+	<para>
+	 Deactivate and delete the &o_meter; proposal before the upgrade, and
+	 re-enable it after the upgrade.
+	</para>
+       </listitem>
+       <listitem>
+	<para>
+	 Deploy the &o_monitor; barclamp before the upgrade.
+	</para>
+       </listitem>
+      </itemizedlist>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>
+     &o_comp;: Xen &compnode;s present
+    </term>
+    <listitem>
+     <para>
+       As of &cloud; 9, the Xen hypervisor is no longer supported. On any
+       cloud, where Xen based compute nodes are still present, the following
+       procedure must be followed before it can be upgraded to &cloud; 9:
+     </para>
+     <itemizedlist>
+    <listitem>
+     <para>
+      Run <command>nova-service disabled</command> on all &o_comp; &compnode;s
```

Comment:
Yikes. I really messed up that part of the document (I checked, the errors are present in my asciidoc text already) now that I look at it again in a different format. There are two things wrong with this:

1. `nova service-disable` (without the `d` at the end and with the dash moved)
2. Run `nova service-disable` _for_ all compute nodes - that command is being run on a controller for every compute node to be disabled.

Reply:
No problem whatsoever! Fixed. :-)